src/content/getting-started/observability/logging/audit-logs/index.md

```diff
@@ -0,0 +1,119 @@
+
+### Kubernetes audit logs
+
+__Warning:__ This feature is currently unavailable on CAPI clusters.
+
+The Kubernetes api-server supports audit logging by default. This mechanism logs every request made to it by both service accounts and users. Customers can ingest these audit logs to detect if any suspicious behaviour (internal or by a 3rd party) is made to their Kubernetes clusters. 
+
+__Example:__
+
+```json
+{
+    "kind": "Event",
+    "apiVersion": "audit.k8s.io/v1",
+    "level": "RequestResponse",
+    "auditID": "a6029022-4ff0-4c54-97ed-4099d0ca1923",
+    "stage": "RequestReceived",
+    "requestURI": "/api/v1/namespaces/default/serviceaccounts?fieldManager=kubectl-create",
+    "verb": "create",
+    "user": {
+        "username": "kubernetes-admin",
+        "groups": ["system:masters", "system:authenticated"]
+    },
+    "sourceIPs": ["172.31.22.88"],
+    "userAgent": "kubectl/v1.23.6 (linux/amd64) kubernetes/ad33385",
+    "objectRef": {
+        "resource": "serviceaccounts",
+        "namespace": "default",
+        "apiVersion": "v1"
+    },
+    "requestReceivedTimestamp": "2022-07-31T08:36:48.679291Z",
+    "stageTimestamp": "2022-07-31T08:36:48.679291Z"
+}
+```
+
+For more information on Kubernetes audit logs, we suggest you to read [this](https://kubernetes.io/docs/tasks/debug/debug-cluster/audit/)
+
+At the moment, the Kubernetes audit logs can be found on `control-plane` nodes only at the location `/var/log/apiserver/audit*.log` that you can ship to your <abbr title="Security information and event management">SIEM</abbr> system.
+
+The Kubernetes audit logs will be shipped to our [Managed Loki](https://github.com/giantswarm/roadmap/issues/311) in the future.
+
+### Machine audit logs
+
+To provide you with a better visibility and security, Giant Swarm offers you the possibility to access the Machine audit logs.
+
+This solution relies on the [Linux Audit Daemon](https://linux.die.net/man/8/auditd) to collect the logs of any execution happening on the machine.
+It is currently configured as such:
+
+```
+-a exit,always -F arch=b64 -S execve -k auditing # -k auditing gives those logs a key so we can easily retrieve them
+-a exit,always -F arch=b32 -S execve -k auditing
+```
+
+This configuration asks the Linux Audit Daemon to trace any `execve` signal happening on the node and renders logs like:
+
+```sh
+Feb 06 09:57:37 ip-10-0-5-34.eu-central-1.compute.internal kernel: audit: type=1300 audit(1675677457.491:6132447): arch=c000003e syscall=59 success=yes exit=0 a0=55a8f8fc7fd0 a1=55a8f947eec0 a2=55a8f9b8fd10 a3=55a8f947eec0 items=2 ppid=1 pid=3659596 auid=4294967295 uid=0 gid=0 euid=0 suid=0 fsuid=0 egid=0 sgid=0 fsgid=0 tty=(none) ses=4294967295 comm="update-ssh-keys" exe="/usr/bin/update-ssh-keys" subj=system_u:system_r:kernel_t:s0 key="auditing"
+```
+
+Beware that the Linux Audit Daemon is quite verbose when it comes to logging so you will need to ensure enough storage and bandwidth for your logs.
```

Comment:
How customers can do that? it would be good to give them a tip

Reply:
TBH, I don't really know. My only suggestions would be to use an existing SIEM, elasticsearch or something with s3 capabilities like Loki. I can add that though :D

Reply:
Done :)

Reply:
Well I meant how they can increase storage. Is this using docker logs volume? can we increase it? (IIRC we can in some providers)

Reply:
Ohh that's machine volume size so that's probably something we would increase for them